core/dbt/contracts/graph/nodes.py

```diff
@@ -1285,27 +1311,76 @@ def same_contents(self, old: Optional["Exposure"]) -> bool:
 # ====================================
 
 
+@dataclass
+class WhereFilter(dbtClassMixin, Replaceable):
```

Comment:
Gerda recommended in my review that we not use Replaceable unless we need it, and I don't think we need it on these classes.